update.py

```diff
@@ -88,11 +90,18 @@ def handle_notices_diff(notices):
     print 'Checking ', len(notices), 'notices'
     for notice in notices:
         sanitised_notice = sanitise_notice_for_database(notice)
-        db_notice = notices_coll.find_one(sanitised_notice)
+        notice_cpy = shallow_copy(sanitised_notice)
```

Comment:
Hey Akshat, can you help me understand why these changes are here?

Reply:
I really don't know why this is showing up here. This is already in the main of the metakgp repo. 😕

Reply:
Okay, no worries.